### ISSUE TITLE
Cannot find module './prebuildTasks'

### ISSUE THREAD

**Issue by JHWu92** [KarlZhu-SE/ark-funds-monitor#143]
Hi KarlZhu,

Thank you for the great app. I am running your project locally to occasionally check ARK's trades. I don't know much about react-app, but I managed to run the app by 1) cloning the project; 2) npm install; 3) npm start until yesterday. I fetch the new update just now and npm start stops working. Some of error outputs are:

Cannot find module './prebuildTasks'
Require stack:
- D:\ark-funds-monitor\config\webpack.config.js
- D:\ark-funds-monitor\scripts\start.js

Can you tell me how to fix this? Thanks!

Jiahui

**Comment by KarlZhu-SE** [comment]
Hi Jiahui,
Thanks for reaching out to me. Yesterday I removed some files since I found another guy copied my repo, put it on their public website with their own donation link on it, and do not follow AGPL-license. This is not the first time, maybe not the last time either.
That is why I decided that this repo will only keep React/front-end code.

If you wanna check the trans, I have it hosted on https://github.com/KarlZhu-SE/ark-funds-monitor.
Sorry for the inconvenience.
